numba/_listobject.h

```diff
@@ -0,0 +1,99 @@
+/* Adapted from CPython3.7 Include/listobject.h */
```

Comment:
This needs a reference URL with an exact SHA from the `python/cpython` github repo: https://github.com/python/cpython

Reply:
fixed with fbd34400b684ca7ad67f3e547a28a5e40a4ffd16